examples/marketplace_image/main.tf

```
@@ -0,0 +1,45 @@
+provider "azurerm" {
+  features {}
+}
+
+module "virtual_machine" {
+  source = "../.."
+
+  virtual_machine_config = {
+    hostname       = "CUSTAPP001"
+    location       = azurerm_resource_group.this.location
+    size           = "Standard_B1ms"
+    os_sku         = "smvm12"
+    os_offer       = "seppmailvirtualmachine"
+    os_version     = "12.0.5"
+    os_publisher   = "seppmailag"
+    severity_group = "01-second-monday-0300-XCSUFEDTG-reboot"
+    enable_plan    = true
+  }
+  admin_username = "local_admin"
+  admin_credential = {
+    admin_password = "H3ll0W0rld!"
+  }
+
+  resource_group_name = azurerm_resource_group.this.name
+  subnet              = azurerm_subnet.this
+}
+
+resource "azurerm_resource_group" "this" {
+  name     = "rg-TestLinuxBasic-tst-01"
```

```suggestion
  name     = "rg-TestLinuxMarketplace-tst-01"
```
Otherwise it might clash when applying more than one example at a time